src/global/global_context.h

```diff
@@ -29,5 +29,6 @@ extern const char *g_assert_condition;
 extern unsigned long long g_assert_thread;
 extern char g_assert_thread_name[4096];
 extern char g_assert_msg[8096];
+extern char g_process_name[4096];
```

Comment:
the maximum filename length is `NAME_MAX`, normally it is 255. which is much smaller than the maximum length of path. so probably, we can use `NAME_MAX` instead?

Reply:
I just used what was for `g_assert_thread_name`, as they looked similar. But I am fine with `NAME_MAX + 1`.